sources/packages/backend/apps/db-migrations/src/sql/DisbursementOverawards/Create-disbursement-overawards-table.sql

```diff
@@ -0,0 +1,45 @@
+CREATE TABLE IF NOT EXISTS sims.disbursement_overawards(
+  id SERIAL PRIMARY KEY,
+  student_id INT NOT NULL REFERENCES sims.students(id),
+  student_assessment_id INT REFERENCES sims.student_assessments(id),
+  disbursement_schedule_id INT REFERENCES sims.disbursement_schedules(id),
+  overaward_value NUMERIC(8, 2) NOT NULL,
```

Comment:
How is the deduction of overaward through disbursement is represented here? 

Also how are deduction of overawards like student paying the money or overaward subtracted from a disbursement represented here.

Reply:
Overaward represents leftovers from the disbursements only. Disbursement amounts are used during the assessment/reassessment of the same application. If a reassessment happens and the student already received more money than they are supposed to receive it will generate an overaward.